spec/u-namespace.md

```diff
@@ -0,0 +1,65 @@
+# MessageFormat 2.0 Unicode Namespace
+
+The `u:` namespace is reserved for use by the Unicode Consortium.
+
+## Options
+
+This section describes common **_<dfn>`u:` options</dfn>_** which each implementation SHOULD support
+for all _functions_ and _markup_.
+
+### `u:id`
+
+A string value that is included as an `id` or other suitable value
+in the formatted parts for the _placeholder_,
+or any other structured formatted results.
+
+Ignored when formatting a message to a string.
+
+Accepts string values, or values which can be stringified without error.
+For other values, a _Bad Option_ error is emitted
+and the `u:id` option is ignored.
+
+### `u:locale`
+
+A comma-delimited list of BCP 47 language tags,
+or an implementation-defined list of such tags.
+
+Replaces the _locale_ defined in the _function context_ for this _expression_.
+The value is ignored when set on _markup_.
+
+During processing, the `u:locale` option
+is always removed from the resolved mapping of _options_.
```

Comment:
Tighten a bit?

```suggestion
is removed from the _resolved options_.
```

Reply:
Two thoughts here:
1. The "always" is intended to ensure that `u:locale` is removed from the options even if this bit from below applies:
   > For unsupported values, a _Bad Option_ error is emitted
   > and the value of the `u:locale` option is ignored.

   Without the "always", is it clear enough that it's always removed?
2. We don't define the term _resolved options_. This is what we have: https://github.com/unicode-org/message-format-wg/blob/e9fc1367cbf496897736e68b0a29fd3ace229759/spec/formatting.md?plain=1#L309-L310

Reply:
WRT (1), I think the always is clear because "is removed" leaves no wiggle room. But I could live with always.

WRT (2), ah... I see. Perhaps:

```suggestion
MUST be removed from the _options_ in the _resolved value_.
```

That's stronger than _is_ and clarifies always. I do find that the phrasing is inelegant though and we could just ignore this comment.

Reply:
Applied some MUSTard.